docs/notes/bugfix-23096.md

```diff
@@ -0,0 +1 @@
+# Implement workaround for AppleScript execution in Rosetta tranalated environment
```

Comment:
Bug fix notes should describe what has been fixed not - in this case I believe it fixes a significant delay when using AppleScript to target some Applications when running on M1 architecture macs?